mlflow/tracking/fluent.py

```diff
@@ -62,7 +62,7 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 
 
 def start_run(run_uuid=None, experiment_id=None, source_name=None, source_version=None,
-              entry_point_name=None, source_type=None, run_name=None):
+              entry_point_name=None, source_type=None, run_name=None, nested=False):
```

Comment:
Could we update the docstring to mention the new `nested` arg?

Reply:
Yup, good catch.